RexNegotumOrganizerCore/src/com/rexnegotium/model/Task.java

```diff
@@ -1,73 +1,72 @@
 package com.rexnegotium.model;
 
-import java.util.Date;
+import java.time.LocalDateTime;
+import java.util.List;
 
 public class Task {
 	
-	private int tid;
-    private String tname;
+    private String name;
     private String note;
-    private String context;
-    private Date begintime;
-    private Date endtime;
+    private List context;
+    private LocalDateTime begindatetime;
+    private LocalDateTime enddatetime;
     private boolean completed;
+    private boolean isFavourite;
+    private User user;
+  //private Project project;
     
-    //private boolean repeatable; // повторение задания, 
-    
-    
-    public int getTid() {
-    	return tid;
-    }
-    public void setTid(int tid) {
-    	this.tid = tid;
-    }
-    
-    public String getTname() {
-    	return tname;
-    }    
-    public void setTname(String tname) {
-    	this.tname = tname;
+    public Task() {
+    		
     }
-    
-    private String getContext() {
+    	
+    	
+	private String getName() {
+		return name;
+	}
+	private void setName(String name) {
+		this.name = name;
+	}
+	private String getNote() {
+		return note;
+	}
+	private void setNote(String note) {
+		this.note = note;
+	}
+	private List getContext() {
 		return context;
 	}
-	private void setContext(String context) {
+	private void setContext(List context) {
 		this.context = context;
 	}
-	public String getNote() {
-    	return note;
-    }
-    public void setNote(String note) {
-    	this.note = note;
-    }
-	private Date getBegintime() {
-		return begintime;
+	private LocalDateTime getBegindatetime() {
+		return begindatetime;
 	}
-	private void setBegintime(Date begintime) {
-		this.begintime = begintime;
+	private void setBegindatetime(LocalDateTime begindatetime) {
```

Comment:
Here and forth Java use camelCase see smth like [here ](https://www.geeksforgeeks.org/java-naming-conventions/) notation like "endTime". 
So setter/getter would be "setEndTime/getEndTime". 
Just FYI and good styling.